struct_env/pymarl_ma_struct_corr.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
```

Comment:
This file should be removed before accepting the PR.
(once everything is done)

Reply:
Agreed! Let's remove duplicated code.